src/Interpreters/executeQuery.cpp

```diff
@@ -171,6 +178,22 @@ static void logQuery(const String & query, const Context & context, bool interna
                 "OpenTelemetry traceparent '{}'",
                 client_info.client_trace_context.composeTraceparentHeader());
         }
+
+        QueryLogElement elem;
```

Comment:
You should not add new records to query_log. Use the existing records.

Reply:
Do you mean we should query for the existing records by query id and modify the existing records?

Reply:
No.

Comment should be added to the QueryLog.h.
Then we will set this field in the place where we already fill QueryLogElement.